### ISSUE TITLE
move www scan status field

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1318]
to after Scan Status - Sitemap.xml and before  Accessibility - Violations

**Comment by gbinal** [comment]
this is done.
